src/helpers.js

```diff
@@ -19,19 +26,41 @@ export const createLink = (href, text) => (
   </a>
 );
 
-export const createEligibilityTooltip = (eligibility) => {
+export const populateEligibilityColumn = (eligibility) => {
   return eligibility.tooltip ? (
-    <Tooltip content={eligibility.tooltip}>
-      <span>{eligibility.title}</span>
-    </Tooltip>
+    <Popover
+      distance={0}
+      position="right"
+      triggerAction="hover"
+      headerContent={
+        <Title headingLevel="h4">
+          <BanIcon className={xsIcon} /> System is not eligible to run this task
+        </Title>
+      }
+      bodyContent={
+        <Flex direction={{ default: 'column' }}>
+          {eligibility.tooltip.split('. ').map((text, index) => (
+            <li key={index}>{text}</li>
+          ))}
```

Comment:
Since there are `li` tags, we should also wrap them with `ul`.

Reply:
Oops, forgot this one.  I'll have to add it later.

Reply:
Done!